src/main/java/roomescape/service/ReservationService.java

```diff
@@ -6,20 +6,28 @@
 import roomescape.dto.ReservationRequest;
 import roomescape.dto.ReservationResponse;
 import roomescape.model.Reservation;
+import roomescape.model.ReservationTime;
 import roomescape.repository.ReservationDao;
+import roomescape.repository.TimeDao;
 
 @Service
 public class ReservationService {
 
     private final ReservationDao reservationDao;
+    private final TimeDao timeDao;
 
-    public ReservationService(final ReservationDao reservationDao) {
+    public ReservationService(final ReservationDao reservationDao, final TimeDao timeDao) {
         this.reservationDao = reservationDao;
+        this.timeDao = timeDao;
     }
 
-    public ReservationResponse save(final ReservationRequest request) {
-        final Reservation created = Reservation.create(request.name(), request.date(), request.time());
-        final Reservation saved = reservationDao.save(created);
+    public ReservationResponse save(final ReservationRequest requestV2) {
```

Comment:
```suggestion
    public ReservationResponse save(final ReservationRequest request) {
```
`V2`는 리팩터링에 사용했다면 제거해주세요.